mon-pix/app/services/current-user.js

```diff
@@ -1,10 +1,12 @@
+import classic from 'ember-classic-decorator';
 import Service, { inject as service } from '@ember/service';
 import _ from 'lodash';
 
-export default Service.extend({
+@classic
```

Comment:
A priori on peut se passer du `@classic`.
D'après la documentation : 
```
In order to remove the classic decorator from a class, you must:

Remove usage of mixins from the class
Remove usage of static class methods on the class, such as reopen and reopenClass
Remove usage of classic class methods within the class definition, including:
get
set
getProperties
setProperties
getWithDefault
incrementProperty
decrementProperty
toggleProperty
addObserver
removeObserver
notifyPropertyChange
```

Reply:
en l'occurrence, ici il y a encore 2 instructions qui sont assimilées à du `@classic` : 
- `this.get('session.isAuthenticated')`
- `this.set('user', user);`

Reply:
oui j'ai raté ma phrase, je voulais dire qu'on peut se passer du `@classic` partout sauf ici

Reply:
d'ailleurs en faisant peu de modifications on peut virer le `@classic`:

```
this.get('session.isAuthenticated'
```
devient
```
this.session.isAuthenticated
```

et

```
const user = await this.store.queryRecord('user', { me: true });
this.set('user', user);
```
devient
```
this.user = await this.store.queryRecord('user', { me: true });

```